### PR TITLE
Re-adds new items to the loadout system

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

A redo of my old PR, this PR adds:

- Glasses
Geist Gazers
Psych Glasses
Thin Glasses
Monocle
- Hats
Maid Headband
- Masks (New!)
Fake Moustache
Italian Moustache
Facescarf
Bandana
Skull Bandana
Party Horn
Cigarette
Cigar
Candy Cigarette
Large Lollipop
- Neck
Bowtie
Maid Neckpiece
- Shoes (New!)
Laced Sandals
Laced Grilling Sandals
Laced Black Sandals
Jockboots
Jack Boots
Swag Shoes
Saints Shoes
- Accessories
Maid Corset
Lipstick Pocket Protector
Armband
## Why It's Good For The Game

More customization that isn't too wild is good and fun.
I've taken some items out of the rotation due to the loadout document's requisites, though some may need to be removed still. It really depends on how much the mantainers want to stretch the definitions.

Removed items include the sad propeller hat (i kept it in the code tho its funny), night caps, gohei/skub, prop masks, balaclava, kindle kicks, skate shoes

Jockboots are laced jackboots. All the sandals are laced.
## Changelog
:cl:
add: Adds a ton of items to the loadout menu, including two new categories of masks & shoes
/:cl:

### CODE REVIEW

File: code/modules/clothing/shoes/sandals.dm [hunk 1]
```diff
@@ -11,14 +11,20 @@
...
+
 /obj/item/clothing/shoes/sandal/alt
 	desc = "A pair of shiny black wooden sandals."
 	name = "black sandals"
 	icon_state = "blacksandals"
 	inhand_icon_state = "blacksandals"
 
-/datum/armor/shoes_sandal
-	bio = 10
+/obj/item/clothing/shoes/sandal/alt/laced
```
Comment (carlarctg) [inline code/modules/clothing/shoes/sandals.dm:25]:
```suggestion
/datum/armor/shoes_sandal
	bio = 10
/obj/item/clothing/shoes/sandal/alt/laced
```

**Comment by carlarctg** [comment]
@Pepsilawn terry player, use your words and express yourself

**Comment by LemonInTheDark** [comment]
I'm going to speak to moth about this. this list sucks and I don't entirely trust you to execute on loadouts properly since either you're trying to work at opposites with the idea of the system or you're just really bad at it.

A brief review, didn't dig into everything just what stood out.

* Glasses
    Geist Gazers No
    Psych Glasses No
    Thin Glasses
    Monocle
* Hats
    Maid Headband Absolutely not
* Masks (New!)
    Fake Moustache
    Italian Moustache I don't like this
    Facescarf
    Bandana
    Skull Bandana Vibes weird but maybe
    Party Horn This sucks let it be unique
    Cigarette
    Cigar ''
    Candy Cigarette
    Large Lollipop ''
* Neck
    Bowtie
    Maid Neckpiece No
* Shoes (New!)
    Laced Sandals
    Laced Grilling Sandals
    Laced Black Sandals
    Jockboots What
    Jack Boots What (again)
    Swag Shoes No
    Saints Shoes No
* Accessories
    Maid Corset Bro what the fuck
    Lipstick Pocket Protector No
    Armband Weird

**Comment by Dawnseer** [comment]
You can craft grilling sandals with wood though

**Comment by mc-oofert** [comment]
echo layla buff pr
also fake moustache hides your identity

**Comment by mc-oofert** [comment]
also you may not add cosmetics to the cosmetics loadout system... 1984

**Comment by carlarctg** [comment]
the cosmeticn't system

**Comment by carlarctg** [comment]
also jockboots are jackboots but laceable and armorless

**Comment by LemonInTheDark** [comment]
I do not trust you with this

**Comment by Mickyan** [comment]
> I'm going to speak to moth about this. this list sucks and I don't entirely trust you to execute on loadouts properly since either you're trying to work at opposites with the idea of the system or you're just really bad at it.
> 
> A brief review, didn't dig into everything just what stood out.
> 
> * Glasses
>   Geist Gazers No
>   Psych Glasses No
>   Thin Glasses
>   Monocle
> * Hats
>   Maid Headband Absolutely not
> * Masks (New!)
>   Fake Moustache
>   Italian Moustache I don't like this
>   Facescarf
>   Bandana
>   Skull Bandana Vibes weird but maybe
>   Party Horn This sucks let it be unique
>   Cigarette
>   Cigar ''
>   Candy Cigarette
>   Large Lollipop ''
> * Neck
>   Bowtie
>   Maid Neckpiece No
> * Shoes (New!)
>   Laced Sandals
>   Laced Grilling Sandals Robbing chef
>   Laced Black Sandals
>   Jockboots What
>   Jack Boots
>   Swag Shoes No
>   Saints Shoes No
> * Accessories
>   Maid Corset Bro what the fuck
>   Lipstick Pocket Protector No
>   Armband Weird

Swanni here, I agree with most of Lemon's list, a few extra that were missed

Fake Moustache: Hides your identity. I'd also add this is the kind of item that serve no cosmetic use except being powergaming bait
Italian Moustache: Same as above
Facescarf: Hides your identity.
Bandana: Hides your identity.
Skull Bandana: Wouldn't you know, hides your identity.
Swirl Lollipop: These have omnizine, happiness and an absurd amount of sugar in them.

The rest seem okay

**Comment by mc-oofert** [comment]
you can hide your identity 10 seconds from arriving with a gasmask or otherwise that is literally no argument against bandanas or the scarves or whatever

**Comment by Dawnseer** [comment]
I will humbly put in a request for a clipboard on loadout

**Comment by EmptyLullaby** [comment]
Respectfully I don't know why you would add all these weird shoes and not even include something basic with no mechanics or memes other than drip like laceup shoes and the like.

**Comment by carlarctg** [comment]
ñ

**Comment by Dawnseer** [comment]
Is that a no on the clipboard?

**Pull request closed without merging**